--- a/desktoppackage/contents/lockscreen/LockScreenUi.qml
+++ b/desktoppackage/contents/lockscreen/LockScreenUi.qml
@@ -144,13 +144,13 @@
             }
         }
         onExited: {
-            uiVisible = false;
+             
         }
         Keys.onEscapePressed: {
             // If the escape key is pressed, kscreenlocker will turn off the screen.
             // We do not want to show the password prompt in this case.
             if (uiVisible) {
-                uiVisible = false;
+                 
                 if (inputPanel.keyboardActive) {
                     inputPanel.showHide();
                 }
@@ -167,7 +167,7 @@
             onTriggered: {
                 if (!lockScreenRoot.blockUI) {
                     mainBlock.mainPasswordBox.showPassword = false;
-                    lockScreenRoot.uiVisible = false;
+                     
                 }
             }
         }
@@ -192,6 +192,7 @@
             from: 0
             to: 1
             duration: Kirigami.Units.veryLongDuration * 2
+            onFinished: lockScreenRoot.uiVisible = true;
         }
 
         Component.onCompleted: launchAnimation.start();
