--- LockScreenUi.qml.orig	2026-06-14 19:32:22.274572496 +1000
+++ LockScreenUi.qml	2026-06-14 19:32:22.282572463 +1000
@@ -107,7 +107,7 @@
     MouseArea {
         id: lockScreenRoot
 
-        property bool uiVisible: false
+        property bool uiVisible: true
         property bool seenPositionChange: false
         property bool blockUI: containsMouse && (mainStack.depth > 1 || mainBlock.mainPasswordBox.text.length > 0 || inputPanel.keyboardActive)
 
@@ -194,7 +194,10 @@
             duration: Kirigami.Units.veryLongDuration * 2
         }
 
-        Component.onCompleted: launchAnimation.start();
+        Component.onCompleted: {
+        launchAnimation.start();
+        authenticator.startAuthenticating();
+    }
 
         WallpaperFader {
             anchors.fill: parent
