--- LockScreenUi.qml.orig	2026-06-14 19:59:54.288365962 +1000
+++ LockScreenUi.qml	2026-06-14 19:59:54.298365928 +1000
@@ -107,7 +107,7 @@
     MouseArea {
         id: lockScreenRoot
 
-        property bool uiVisible: false
+        property bool uiVisible: true
         property bool seenPositionChange: false
         property bool blockUI: containsMouse && (mainStack.depth > 1 || mainBlock.mainPasswordBox.text.length > 0 || inputPanel.keyboardActive)
 
@@ -144,13 +144,13 @@
             }
         }
         onExited: {
-            uiVisible = false;
+             
         }
         Keys.onEscapePressed: {
             // If the escape key is pressed, kscreenlocker will turn off the screen.
             // We do not want to show the password prompt in this case.
             if (uiVisible) {
-                uiVisible = false;
+                 
                 if (inputPanel.keyboardActive) {
                     inputPanel.showHide();
                 }
@@ -167,7 +167,7 @@
             onTriggered: {
                 if (!lockScreenRoot.blockUI) {
                     mainBlock.mainPasswordBox.showPassword = false;
-                    lockScreenRoot.uiVisible = false;
+                     
                 }
             }
         }
@@ -194,7 +194,10 @@
             duration: Kirigami.Units.veryLongDuration * 2
         }
 
-        Component.onCompleted: launchAnimation.start();
+        Component.onCompleted: {
+        launchAnimation.start();
+        authenticator.startAuthenticating();
+    }
 
         WallpaperFader {
             anchors.fill: parent
@@ -202,7 +205,7 @@
             source: wallpaper
             mainStack: mainStack
             footer: footer
-            clock: clock
+            clock: clock; visible: false
             alwaysShowClock: config.alwaysShowClock && !config.hideClockWhenIdle
         }
 
