--- Main.qml	2026-06-14 19:24:14.208514602 +1000
+++ Main.qml	2026-06-14 19:24:14.212514587 +1000
@@ -102,15 +102,7 @@
                 }
             }
         }
-        WallpaperFader {
-            visible: config.type === "image"
-            anchors.fill: parent
-            state: loginScreenRoot.uiVisible ? "on" : "off"
-            source: wallpaper
-            mainStack: mainStack
-            footer: footer
-            clock: clock
-        }
+        
 
         DropShadow {
             id: clockShadow
