--- Main.qml	2026-06-14 19:59:54.290365955 +1000
+++ Main.qml	2026-06-14 20:00:13.824299501 +1000
@@ -98,19 +98,11 @@
             onTriggered: {
                 if (!loginScreenRoot.blockUI) {
                     userListComponent.mainPasswordBox.showPassword = false;
-                    loginScreenRoot.uiVisible = false;
+                     
                 }
             }
         }
-        WallpaperFader {
-            visible: config.type === "image"
-            anchors.fill: parent
-            state: loginScreenRoot.uiVisible ? "on" : "off"
-            source: wallpaper
-            mainStack: mainStack
-            footer: footer
-            clock: clock
-        }
+        
 
         DropShadow {
             id: clockShadow
